example/index.ts

```diff
@@ -1,9 +1,23 @@
-import { Listener } from '../lib/';
+import { Context } from '../lib/context';
+import { Config, Emitter, Event, Listener } from '../lib/';
 
-/**
- * with rabbit mq running, and ready to publish for exchange => RABBIT_EVENT_PEOPLE_TOPIC_NAME
- */
-const eventName = 'message.*.schedule';
-Listener.on(eventName, (event: Record<string, any>) => {
-	console.log('resolved event', event);
-});
+(async () => {
+	await new Config().init();
+	const eventName = 'message.*.schedule';
+	Listener.on(eventName, (event: Event, context: Context) => {
+		console.log('consume event');
+		console.log(event.payload());
+		context.success(event);
+	});
+
+	const events: Event[] = [
+		new Event('message.*.schedule', {
+			msg: 'hello goddess world',
+		}),
+		new Event('message.*.schedule', {
+			msg: 'hello goddess world 2',
+		}),
+	];
+
+	Emitter.trigger(events);
+})();
```

Comment:
Same as for the Daemon example but also, this file should be renamed to `listener.ts`.